packages/offix-client/src/apollo/ApolloOfflineClient.ts

```diff
@@ -1,8 +1,22 @@
 import { NormalizedCacheObject } from "apollo-cache-inmemory";
 import { ApolloClient, OperationVariables, MutationOptions } from "apollo-client";
-import { OfflineStore, OfflineQueueListener, OfflineQueue } from "offix-offline";
-import { MutationHelperOptions } from "offix-cache";
 import { FetchResult } from "apollo-link";
+import {
+  OfflineStore,
+  OfflineQueueListener,
+  OfflineQueue,
+  IResultProcessor,
+  QueueEntryOperation,
+  QueueEntry
+} from "offix-offline";
+import { MutationHelperOptions } from "offix-cache";
+
+export type ApolloOfflineQueue = OfflineQueue<MutationOptions>;
```

Comment:
It will be cool to get some minimum docs for those.

Reply:
Okay going to do that before merging